### ISSUE TITLE
filter no longer working with vimeo

### ISSUE THREAD

**Issue by bee911** [FrostCo/AdvancedProfanityFilter#504]
hello, last year you helped getting filtering working with my finance site which uses vimeo for videos.  it has worked perfectly but for some resaon when i enable CC as always did, in vimeo video player that is on website all it is doing is displaying the CC text and never filtering.  when i click profanity filter icon while on site it says audio muting active and i can see it found words but those word were not muted

hereis customization you provide me long ago.

```json
{
  "player.vimeo.com": [
    {
      "className": "vp-captions-line",
      "displaySelector": "div.vp-captions > span.vp-captions-window",
      "mode": "element",
      "note": "Only works if captions are only on a single line",
      "tagName": "SPAN"
    }
  ],
  "vimeo.com": [
    {
      "className": "vp-captions-line",
      "displaySelector": "div.vp-captions > span.vp-captions-window",
      "mode": "element",
      "note": "Only works if captions are only on a single line",
      "tagName": "SPAN"
    }
  ],
  "web.microsoftstream.com": [
    {
      "displaySelector": "",
      "iframe": false,
      "mode": "watcher",
      "subtitleSelector": "div.video-player-container div.vjs-text-track-display > div > div > div"
    }
  ]
}
```

thank you again for help

**Comment by richardfrost** [comment]
Hi @bee911, thanks for reaching out about this issue. It looks like they have changed the structure of their captions, at least for vimeo.com.

The following config seems to work for me on this test video (filtering test words like "internet" and "videos"):

Video: https://vimeo.com/358296408

```json
{
  "vimeo.com": [
    {
      "displaySelector": "div.vp-captions",
      "mode": "watcher",
      "subtitleSelector": "div.vp-captions span.vp-captions-line"
    }
  ],
  "player.vimeo.com": [
    {
      "displaySelector": "div.vp-captions",
      "mode": "watcher",
      "subtitleSelector": "div.vp-captions span.vp-captions-line"
    }
  ]
}
```

Without seeing the page structure for "web.microsoftstream.com" I probably can't fix that. If there is a publicly available video using that site please send it to me, but if not, maybe you can give me a screenshot of the elements on the page?

**Comment by bee911** [comment]
thank you mr. frost.  that fixed the CC text from showing again on vimeo.  take care.  jlr

**Comment by richardfrost** [comment]
This has been included in version [3.2.2](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/3.2.2). Feel free to comment here if you have any feedback for this update.
